cmd/release/cmd/update.go

```diff
@@ -130,11 +132,44 @@ var updateChartsCmd = &cobra.Command{
 	},
 }
 
+var updateRancherCmd = &cobra.Command{
+	Use:   "rancher",
+	Short: "Update rancher files",
+}
+
+var updateRancherDashboardCmd = &cobra.Command{
+	Use:   "dashboard [version]",
+	Short: "Update Rancher's Dashboard and UI references and create a PR",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
```

Comment:
Let's try to use the standard ValidationArgs function of cobra for validating args?
https://github.com/rancher/ecm-distro-tools/blob/master/cmd/release/cmd/push.go#L47

Reply:
Moved the validation to `Args`.